diesel/src/pg/types/numeric.rs

```diff
@@ -153,7 +152,6 @@ mod bigdecimal {
 
     impl FromSql<Numeric, Pg> for BigDecimal {
         fn from_sql(numeric: Option<&[u8]>) -> deserialize::Result<Self> {
-            // FIXME: Use the TryFrom impl after Rust 1.34 get released.
```

Comment:
Shouldn't this actually do what this comment says? We can get rid of that function and just put the body in the `TryFrom` impl

Reply:
Right, didn't read the code, sorry 😥